big_tests/tests/metrics_api_SUITE.erl

```diff
@@ -47,15 +49,14 @@ all() ->
                        ]).
 
 groups() ->
-    G = [
+    [
          {metrics, [], ?METRICS_CASES},
```

Comment:
Not sure I got it. The structure is correct. What you mean - remove indent? 
This can be 
```erlang
[a, b, c]
```
or 
```erlang
[
    a,
    b,
    c
]
```
I was use the same code alignment what is used in the code.

Reply:
Could you please provide example what you expected?

Reply:
I saw that in many places we are using something like this
```
[
 a,
 b,
 c
]
```
For example `all` function from this file
```
all() ->
    [
     {group, metrics},
     {group, all_metrics_are_global},
     {group, global}
    ].
```

Reply:
Updated.